pmd-swift/src/main/antlr4/net/sourceforge/pmd/lang/swift/antlr4/Swift.g4

```diff
@@ -1078,6 +1078,13 @@ fragment MultiQuotedTextItem : MultiInterpolatedString
  ;
 fragment MultiInterpolatedString: '\\(' (MultiQuotedTextItem | SingleStringLiteral)* ')';
 
+DashedSingleStringLiteral : '#"' DashedMultiQuotedText? '"#' ;
```

Comment:
One may use an arbitrary number of `#` delimiters provided they're balanced, eg `##"..."##`

[This link](https://stackoverflow.com/questions/36896378/how-to-write-a-lexer-rule-that-references-a-character) may be a good starting point to implement this